src/coinbase/webhook.ts

```diff
@@ -163,15 +163,21 @@ export class Webhook {
   }
 
   /**
-   * Updates the webhook with a new notification URI.
+   * Updates the webhook with a new notification URI, and optionally a new list of addresses to monitor.
    *
-   * @param notificationUri - The new URI for webhook notifications.
+   * @param options - The options to update webhook.
+   * @param options.notificationUri - The new URI for webhook notifications.
+   * @param options.eventTypeFilter - The new eventTypeFilter that contains a new list (replacement) of addresses to monitor for the webhook.
    * @returns A promise that resolves to the updated Webhook object.
    */
-  public async update(notificationUri: string): Promise<Webhook> {
+  public async update({
+    notificationUri,
+    eventTypeFilter,
+  }: UpdateWebhookOptions): Promise<Webhook> {
```

Comment:
Just a note that we're breaking back compat by changing the params here. Not sure if this would be an issue at this point, though.

Reply:
this is a good call out cc @jazz-cb not sure if it matters at this point

or maybe we can make the param like this: `notificationUriOrOptions: string | UpdateWebhookOptions`
then we can check their input: `typeof notificationUriOrOptions === 'string'`